### PR TITLE
Work around AttributeError raised by Python 3.5.x.

### PR DESCRIPTION
**Pull request by tseaver**
Alternative to PR #55.

h/t to @jamadden for the patch.

### CODE REVIEW

File: BTrees/BTreeModuleTemplate.c [hunk 1]
```diff
@@ -558,8 +558,15 @@ module_init(void)
     cPersistenceCAPI = (cPersistenceCAPIstruct *)PyCObject_Import(
                 "persistent.cPersistence", "CAPI");
 #endif
-    if (cPersistenceCAPI == NULL)
+    if (cPersistenceCAPI == NULL) {
+       /* At least some versions of Python 3.5 raise an AttribteError
+        * an ImportError here:  re-map as an ImportError for compatibility.
+        */
+       if (PyErr_Occurred() && !PyErr_ExceptionMatches(PyExc_ImportError)) {
+           PyErr_SetString(PyExc_ImportError, "<No CPersistence>");
```
Comment (jamadden) [inline BTrees/BTreeModuleTemplate.c]: `<No CPersistence>` was intended as a placeholder, I was hoping somebody could come up with something better :) Maybe "Unable to import persistent.cPersistence C extension. Re-install persistent without PURE_PYTHON."?

Comment (jamadden) [inline BTrees/BTreeModuleTemplate.c]: If you'd like, I can update the text string on this branch.

Comment (tseaver) [inline BTrees/BTreeModuleTemplate.c]: @jamadden Sure, go ahead.

**Comment by jamadden** [comment]
Travis is green. Would you like to take a look @tseaver?

**Comment by tseaver** [comment]
@jamadden Thanks for the review / fix.

**Pull request merged**